colossalai/fx/profiler/opcount.py

```diff
@@ -201,6 +202,8 @@ def zero_flop_jit(*args):
     # normalization
     aten.native_batch_norm.default: batchnorm_flop_jit,
     aten.native_batch_norm_backward.default: batchnorm_flop_jit,
+    aten.cudnn_batch_norm.default: batchnorm_flop_jit,
+    aten.cudnn_batch_norm_backward.default: partial(batchnorm_flop_jit, training=True),
```

Comment:
notice that cudnn_batch_norm_backward is different because it is only used when `training=True`